INSTALL.rst

```diff
@@ -84,17 +84,40 @@ Installing from source
 
 If you are interested in contributing to Matplotlib development,
 running the latest source code, or just like to build everything
-yourself, it is not difficult to build Matplotlib from source.  Grab
-the latest *tar.gz* release file from `the PyPI files page
-<https://pypi.org/project/matplotlib/>`_, or if you want to
-develop Matplotlib or just need the latest bugfixed version, grab the
-latest git version, and see :ref:`install-from-git`.
+yourself, it is not difficult to build Matplotlib from source.
+
+The easiest way to get the latest development version to start contributing
+is to go to the git `repository <https://github.com/matplotlib/matplotlib>`_
+and run: ::
+
+  git clone https://github.com/matplotlib/matplotlib.git
 
 Matplotlib can be installed from the source directory with a simple ::
 
-  python -m pip install .
+  pip install .
+
+If you're developing, it's better to do it in editable mode. The reason why
+is that there's a few artifacts for testing that are only properly referenced
```

Comment:
```suggestion
is that pytest's test discovery only works for Matplotlib
```

This wording is not great still, but I think we can be explicit that the issue is with pytest test discovery.